policyengine_us/parameters/gov/irs/credits/estate/base.yaml

```diff
@@ -0,0 +1,32 @@
+description: The IRS provides this base exclusion amount under the estate tax credit. 
+metadata:
+  unit: currency-USD
+  label: Estate tax credit base exclusion amount  
+  period: year
+  reference:
+    - title: 26 U.S. Code § 2010 - Unified credit against estate tax - (c)(3)(A)
+      href: https://www.law.cornell.edu/uscode/text/26/2010
+    - title: What's New - Estate and Gift Tax - Basic Exclusion Amount for Year of Death
+      href: https://www.irs.gov/businesses/small-businesses-self-employed/whats-new-estate-and-gift-tax
+    - title: Instructions for Form 706 (Rev. September 2023)
```

Comment:
could we add the same form for 2022 and 2021 as well as a reference that shows the table